content/en/plugins/github-api/use.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Use the GiHub API Plugin
+linkTitle: Use
+weight: 5
+description: >
+  Learn how to use the GitHup API Plugin to trigger Spinnaker pipelines from GitHub and also to trigger GitHub workflows from Spinnaker pipelines.
+---
+
+## {{% heading "prereq" %}}
+
+You should be familiar with [GitHub workflows](https://docs.github.com/en/actions/using-workflows/about-workflows).
+
+## Trigger GitHub workflows from Spinnaker pipelines
+
+Use the **Github API Workflow Trigger Stage** to trigger your GitHub workflow from your Spinnaker pipeline.
+
+### `workflow_dispatch` event
+
+Configure the **Github API Workflow Trigger Stage** as in the following screenshot: 
+
+{{< figure src="/images/plugins/github/workflowDispatch.png" >}}
+
+* **Github Account**: (Required) Select the GitHub Account; this is one of the accounts you configured when you installed the plugin
+* **Organization or User**: (Required) The organization or username that should trigger the workflow
+* **Project**: (Required) Spinnaker project name
+* **Workflow Name**: (Required) The filename of your workflow
+* **Git reference (branch or tag name)**: (Required) The branch or tag name that contains the workflow 
```

Comment:
`**Git reference (branch or tag name)**: (Required) The branch or tag name that contains the workflow ` should be changed to:
`**Git reference (branch or tag name)**: (Required) The branch or tag name that will receive the workflow dispatch event`